lms/db/_columns.py

```diff
@@ -0,0 +1,23 @@
+import sqlalchemy as sa
```

Comment:
If this is for public export then I think our standard is to have this file be called `columns.py` to show it contains public things (even if they are not directly exported from here).